### ISSUE TITLE
Calling bucket.file.exists always returns false

### ISSUE THREAD

**Issue by VanPaitin** [fsouza/fake-gcs-server#1556]
```node
import { Storage } from '@google-cloud/storage';

const storage = new Storage();
const apiBucket = storage.bucket('api-video-song-renderer-bucket-test');
apiBucket.file(`shakaraSTART`).save("Hello world", {
  public: true
});
```
Running the code above correctly creates a file in the bucket. When I check the logs, I see it makes a call to
`POST /upload/storage/v1/b/api-video-song-renderer-bucket-test/o?name=shakaraSTART&uploadType=resumable&predefinedAcl=publicRead`.

However, when I try to check with:

```node
apiBucket.file('shakaraSTART').exists().then(exists => {
  expect(exists).toBe(true);
});
```

I am always getting a false value meaning the file doesn't exist when it infact exists. I checked the logs and it was making calls to
`GET /b/api-video-song-renderer-bucket-test/o/shakaraSTART`

Observe that it doesn't prepend the endpoint with `/storage/v1`. This is probably why I am getting a 404 error.

1. Is there a work around for this?
2. Is this a bug and will it be fixed?
3. What may I be getting wrong?

**Comment by fsouza** [comment]
How did you configure the node client to talk to fake-gcs-server?

**Comment by VanPaitin** [comment]
Thanks @fsouza for your reply. I am configuring the client simply by setting the env variable: STORAGE_EMULATOR_HOST='http://localhost:4443'

**Comment by VanPaitin** [comment]
@fsouza, should I expect a solution?

**Comment by fsouza** [comment]
Can you provide exact steps for reproducing the issue.

I just tried to reproduce with the example in the repo and couldn't: https://github.com/fsouza/fake-gcs-server/pull/1585

If the client is not sending the correct request to the server, I assume the client is misconfigured.

**Comment by fsouza** [comment]
Merged #1585, see that for an example on how to setup the Node client in a way where that method works.
